src/sql/src/plan/statement/show.rs

```diff
@@ -240,29 +240,28 @@ fn show_tables<'a>(
     from: Option<UnresolvedObjectName>,
     filter: Option<ShowStatementFilter<Raw>>,
 ) -> Result<ShowSelect<'a>, anyhow::Error> {
-    if extended {
-        unsupported!("SHOW EXTENDED TABLES");
-    }
-
     let schema = if let Some(from) = from {
         scx.resolve_schema(from)?
     } else {
         scx.resolve_default_schema()?
     };
 
-    let query = if full {
-        format!(
+    let mut query = format!(
```

Comment:
In order for us to grab system tables, we'll need to join with `mz_schemas` like we do in `show_types`:

```
        SELECT t.name, mz_internal.mz_classify_object_id(t.id) AS type
        FROM mz_catalog.mz_types t
        JOIN mz_catalog.mz_schemas s ON t.schema_id = s.id
        WHERE t.schema_id = {}",
        schema.id(),
```
The `s` in your `extended` case is referring to `mz_catalog.mz_schemas`, but it's not accessible yet! To update that, and to match `show_types`, I think you can replace lines 249 - 263 with:
```
    let mut query = format!(
        "SELECT t.name, mz_internal.mz_classify_object_id(t.id) AS type
        FROM mz_catalog.mz_tables t
        JOIN mz_catalog.mz_schemas s ON t.schema_id = s.id
        WHERE schema_id = {}",
        schema.id(),
    );
    if extended {
        query += " OR s.database_id IS NULL";
    }
    if !full {
        query = format!("SELECT name FROM ({})", query);
    }
```

Reply:
Hi @JLDLaughlin, thanks for reviewing my PR. I've added my changes below. How do I re-run the build checks?